scripts/k8s-galera

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+# [This file is part of the zabbix-agent-osso package]
+# REQUIRES: jq, kubectl
+#
+
+get_contexts() {
+    kubectl config get-contexts -oname
+}
+
+get_namespaces() {
+    kubectl --context "$1" get namespace \
+        -l ossobv/zabbix-agent-osso.k8s-galera=true \
+        -o go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' \
+        2>/dev/null
+}
+
+get_clusters() {
+    kubectl --context "$1" -n "$2" \
+        get mariadbs.k8s.mariadb.com -oname 2>/dev/null |
+        cut -d'/' -f2
+}
+
+make_json_clusters() {
+    echo '{"{#CONTEXT}":"'$1'","{#NAMESPACE}":"'$2'",'"\
+        "'"{#CLUSTER}":"'$3'"}'
+}
+
+make_json_cluster_status() {
+    json=$(kubectl --context "$1" -n "$2" get mariadbs.k8s.mariadb.com "$3" \
+        -o json |
+            jq -c '{
+                current_primary_pod_index: .status.currentPrimaryPodIndex,
+                current_replica_count: .status.replicas,
+                galera_ready: (.status.conditions[] |
+                    select(.reason == "GaleraReady") | .status),
+                desired_replica_count: .spec.replicas,
+                sts_ready: (.status.conditions[] |
+                    select(.reason == "StatefulSetReady") | .status)
+            }')
```

Comment:
Does jq send anything to stderr? If so, it should be redirected to /dev/null.

Reply:
I haven't seen it output anything to stderr, while I was testing this

Reply:
okay